src/lib/bootvar.c

```diff
@@ -38,6 +38,11 @@
  * enough. actual space occupied is normally >2 times less. */
 #define BOOT_VAR_MAX 1024
 
+/* env var to turn on the test mode. if set to "yes", turns off any side-effects
+ * functions defined in this file may have: the call will always succeed without
+ * side effects. */
+#define CBM_BOOTVAR_TEST_MODE_VAR "CBM_BOOTVAR_TEST_MODE"
```

Comment:
Could we eventually get a vtable setup so mocking bootvar_* functions can be inserted in tests so we can add code coverage for a few of the functions that don't touch hardware? (Future todo perhaps).